src/UILayer/PacketWindow.cpp

```diff
@@ -0,0 +1,921 @@
+#include "PacketWindow.h"
```

Comment:
This one file definitely has a lot of stuff in it. Split it up into multiple files for each tab, and combine them together!